core-web/apps/dotcms-ui/src/app/portlets/dot-pages/dot-pages-favorite-panel/dot-pages-favorite-panel.component.scss

```diff
@@ -86,6 +87,35 @@
             margin-bottom: $spacing-3;
         }
     }
+
+    .pi-star-fill {
+        color: $color-palette-primary;
+        margin-right: $spacing-1;
+    }
+
+    .dot-pages-panel__header {
+        display: flex;
+        align-items: center;
+        margin-left: $spacing-3;
+        font-weight: 400;
```

Comment:
We have a global variable for this, you can check this folder `core-web/libs/dotcms-scss` there we have all the styles we use and this one you can find it on `core-web/libs/dotcms-scss/shared/_fonts.scss` it is exactly `$font-weight-regular-bold`